apps/web/vibes/soul/primitives/accordions/index.tsx

```diff
@@ -4,6 +4,27 @@ import * as AccordionsPrimitive from '@radix-ui/react-accordion';
 import { clsx } from 'clsx';
 import React from 'react';
 
+/**
+ * This component supports various CSS variables for theming. Here's a comprehensive list, along
+ * with their default values:
+ *
+ * ```css
+ * :root {
+ *   --accordion-light-title-text: hsl(var(--contrast-400));
+ *   --accordion-light-title-text-hover: hsl(var(--foreground));
+ *   --accordion-light-title-icon: hsl(var(--contrast-500));
+ *   --accordion-light-title-icon-hover: hsl(var(--foreground));
+ *   --accordion-light-content-text: hsl(var(--foreground));
+ *   --accordion-dark-title-text: hsl(var(--contrast-200));
+ *   --accordion-dark-title-text-hover: hsl(var(--background));
+ *   --accordion-dark-title-icon: hsl(var(--contrast-200));
+ *   --accordion-dark-title-icon-hover: hsl(var(--background));
+ *   --accordion-dark-content-text: hsl(var(--background));
+ *   --accordion-title-font-family: var(--font-family-mono);
+ *   --accordion-content-font-family: var(--font-family-body);
+ * }
+ * ```
+ */
```

Comment:
Should we add this to the docs but not the component? Same for the rest.

Reply:
This is to help with tooltips in the IDE but mainly to give @agurtovoy / @fikrikarim guidance on the CSS variables to include in the updated `Theme` component. This is how @migueloller wanted to handle it.

Reply:
@jorgemoya, FWIW, I think it should be in both and ideally we drive some of the docs content from strong JSDoc comments in our components and type definitions. I personally like to keep documentation as close to the code as possible and to generate it when possible.